index.js

```diff
@@ -29,13 +29,17 @@ function PocketSphinxContinuous(config) {
 
 PocketSphinxContinuous.prototype.start = function() {
   if (!this.isListening()) {
+    let mic=this.microphone;
+    // if name starts with number
+    if (this.microphone.match(/^\d/))
```

Comment:
Can you please add brackets to the if statement, I'm not a fan of this syntax.

Reply:
In the default config in the repo the microphone is a number, here you assume it is a string. But then the code will fail due to `TypeError: microphone.match is not a function`. See https://jsfiddle.net/38nwxmqs/

Reply:
I fixed it on your branch